code/modules/library/lib_items.dm

```diff
@@ -222,6 +222,10 @@
 	var/unique = FALSE //false - Normal book, true - Should not be treated as normal book, unable to be copied, unable to be modified
 	var/title //The real name of the book.
 	var/window_size = null // Specific window size for the book, i.e: "1920x1080", Size x Width
+	/// Minimum icon state number
+	var/minimum_book_state = 1
```

Comment:
Does this var really need to exist?

Reply:
You're right, I was thinking of a different bit of code when I did that.
I'll fix it